config/config.go

```diff
@@ -271,6 +271,7 @@ type StdoutLoggingOutlet struct {
 
 type SyslogLoggingOutlet struct {
 	LoggingOutletCommon `yaml:",inline"`
+	Facility            string `yaml:"facility,default=local0"`
```

Comment:
hm, sorry, maybe I was unclear: I would like to see the `syslog.Facility` here. that should be possible by defining a wrapper type and implementing `UnmarshalYAML` for it.